marklogic-data-hub-central/src/main/java/com/marklogic/hub/central/auth/WebSecurityConfiguration.java

```diff
@@ -71,6 +71,7 @@ protected void configure(HttpSecurity http) throws Exception {
      */
     protected String[] getAlwaysPermittedPatterns() {
         return new String[]{
+            "/websocket/**",
```

Comment:
I feel it'd be safer for this to be explicitly named for its purpose today, which is for checking session status, right? So this could be "/websocket/sessionStatus", and we'd want to explicitly list that here instead of allowing anything under "/websocket/" (as we could add another path under "/websocket/" that does require authentication and forget to update this).

Reply:
There are a lot more endpoints called for setting up WebSockets and they include randomly generated paths, so determining all the patterns that should be permitted isn't really practical.

I was able to remove the WebSockets permitted pattern by refactoring the front-end to not attempt to connect to WebSockets when logged out, which was bring the browser its knees with connection attempts.